libs/utils/android/workload.py

```diff
@@ -92,7 +92,7 @@ def availables(target):
                     module_name = module.__name__
                     # Check if a package is required and is available on target
                     aw = Workload._AW(module_name, module, class_name, handler)
-                    if (Workload._is_available(target, aw)):
```

Comment:
Still I don't remember the purpose and it's not easy to figure it out from the code.
We should:
1. use better var names, let's redefined:
```python
AndroidWorkload = namedtuple('AndroidWorkload',  ['module_name', 'module', 'class_name', 'ctor'])
```
2. add a better comment which explain why we need that check